src/Book.js

```diff
@@ -0,0 +1,82 @@
+import React, {useState, useEffect} from 'react';
+import './App.css';
+import {loadJson, sendJson} from './sendJson';
+import {useParams} from 'react-router-dom';
+
+const loadOneBook = async (id, onLoad, setError) => {
+  const result = await loadJson('/getOneBook/' + id);
+  if (result.error) {
+    setError(result.error);
+  } else {
+    onLoad(result);
+  }
+};
+
+export const Book = () => {
+  const [bookState, setBookState] = useState([]);
+  const [error, setError] = useState('');
+  const [titleState, setTitleState] = useState('');
+  const {id} = useParams();
+
+  // This is breaking the server. Need to figure out why.
+  // const changeTitle = async () => {
+  //   const result = await sendJson('editBook', {
+  //     newTitle: titleState,
+  //     id
+  //   });
+  //   if (result.error) {
+  //     setError(result.error);
+  //   }
+  //   await loadOneBook(id, setBookState, setError);
+  //   setTitleState('');
+  // };
+
+  useEffect(() => {
+    loadOneBook(id, setBookState, setError);
+  }, []);
+
+  return (
+    <div className="container">
+      <h2>Edit Book Details:</h2>
+
+      <h3 style={{color: 'red'}}>{error}</h3>
+      <div className="container">
+        {bookState.map(
```

Comment:
If it's just one book, we should not need to map over it. Once you fix the server to return an object instead of an array containing an object, you will want to stop using map here.